### PR TITLE
Softkinetic substitute package

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

File: softkinetic_camera/bin/softkinetic_bringup_node [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+echo "this is just a placeholder"
```
Comment (mgruhler) [inline softkinetic_camera/bin/softkinetic_bringup_node]: Nice :-)

Maybe a note to install from source and which node this is?

**Comment by mgruhler** [comment]
CR: ok

**Comment by benmaidel** [comment]
i would prefer a solution where the user get's informed over a missing dependency during compile time and not run time!

**Comment by mgruhler** [comment]
@ipa-bnm then please propose a solution at ipa320/softkinetic

**Comment by benmaidel** [comment]
all right, will do.

**Comment by benmaidel** [comment]
hmpf, just played a bit around. Nothing worked. So no better idea than substituting...

**Comment by benmaidel** [comment]
FR: OK

**Comment by mgruhler** [comment]
@ipa-bnm also pretty wild and just a draft, but what do you think of https://github.com/ipa-mig/softkinetic/tree/dummy_node

**Comment by benmaidel** [comment]
@ipa-mig i had nearly the exact same solution running :) To fool Jenkins, it think this is ok. Just one Remark: shall we print periodic ROS_ERRORs if the sdk is not installed?

**Comment by mgruhler** [comment]
@ipa-fmw @ipa-nhg your thoughts?

**Comment by NadiaHG** [comment]
@ipa-mig @ipa-bnm great solution! :+1: Thanks

can you please open a new pull request to check if travis is able to compile the `softkinetic_camera` package?

**Comment by mgruhler** [comment]
@ipa-nhg Well, this is yet just a draft, we need to write this properly :-) But if this is something that you feel is okay, we could probably do this.

However, we should not release it like this!

**Comment by benmaidel** [comment]
this was my solution: https://github.com/ipa-bnm/softkinetic/commit/c9561d0eb778f0124f4b279add2a9110c378f696, just to be complete.
nearly the same as @ipa-mig solution, only that here a ros node with periodic error messages is generated.

**Comment by mgruhler** [comment]
I like @ipa-bnm solution more.

**Comment by NadiaHG** [comment]
Thanks @ipa-bnm,  good solution! this pull request to cob_substitute can be closed.

**Comment by benmaidel** [comment]
opened new pr: https://github.com/ipa320/softkinetic/pull/59

**Comment by floweisshardt** [comment]
as https://github.com/ipa320/softkinetic/pull/59 is merged: can we close this PR without merging?

**Pull request closed without merging**